README.md

```diff
@@ -8,6 +8,15 @@ suitable for internal use only at this time.
 
 Install one or more of the `RSpec` gems.
 
+Want to run against the `master` branch? You'll need to include the dependent
+RSpec repos as well. Add the following to your `Gemfile`:
+
+```ruby
+%w[rspec-core rspec-expectations rspec-mocks rspec-support].each do |lib|
+  gem lib, :git => "git://github.com/rspec/#{lib}.git", :branch => 'master'
+end
+```
+
```

Comment:
I think it's less likely that people are gonna be aware of needing to do this for rspec-support since it doesn't have any public APIs.  I'm fine including this, though.